src/lexer.coffee

```diff
@@ -900,7 +907,7 @@ CODE       = /^[-=]>/
 
 MULTI_DENT = /^(?:\n[^\n\S]*)+/
 
-JSTOKEN    = /^`[^\\`]*(?:\\.[^\\`]*)*`/
+JSTOKEN    = /^```([\s\S]*?)```|^`[^\\`]*(?:\\.[^\\`]*)*`/
```

Comment:
I read the added regex part as "greedily capture everything between starting and end triple backquotes, which seems sane.

The original regex is rather obtuse. I read it as "greedily read everything between starting and ending single backquotes, and make sure every backslash has a character following it". I'm thinking the backslash+character part exists to stop an escaped backslash preemptively closing the Javascript block. E.g:
```
OK: `a\``
Not OK: `a\`
```

The new triple backticks regex will behave differently:

```
OK: ```a\````
Also OK: ```a````
```

I found this website useful: https://regex101.com/